app/routes/application/review.js

```diff
@@ -0,0 +1,118 @@
+const utils = require('./../../utils')
+
+/**
+ * Application: Review answers
+ */
+module.exports = router => {
+  router.get('/application/:applicationId/review', (req, res) => {
+    var applicationData = utils.applicationData(req)
+    var pageObject = {}
+    var successFlash = req.flash('success')
+
+    if (successFlash[0] === 'submitted-incompleted-application') {
+      pageObject.errorList = []
+      const sections = {
```

Comment:
This now looks like if the user submits an incomplete application then every single thing is marked as incomplete instead of just the incomplete things.

Reply:
With the changes made so far, behaviour appears correct; the errors shown at the top of the page reflect the incomplete sections. So may skip refactoring this.